frontend/src/app/components/nav-bar/nav-bar.component.html

```diff
@@ -0,0 +1,15 @@
+<nav class="app-navbar navbar">
+    <a class="app-navbar-brand" [routerLink]="'/' + homeRoute">
+        <ng-container *ngIf="showPageTitle; else app_brand">
+            <mat-icon class="app-navbar-brand__icon">arrow_back</mat-icon>
+            <span class="app-navbar-brand__text">{{activeRoute}}</span>
+        </ng-container>
+        <ng-template #app_brand >
+            <img class="app-navbar-brand__logo" src="/assets/icon/medtagger-logo.svg" alt="Medtagger logo" />
```

Comment:
Am I the only one with empty logo here?
<img width="283" alt="screenshot 2018-11-11 at 21 36 45" src="https://user-images.githubusercontent.com/6514553/48317992-05ff4400-e5fa-11e8-96b7-7ebe5fcd71d6.png">

This should probably fix it:
```suggestion
            <img class="app-navbar-brand__logo" src="/assets/icon/medtagger_logo.svg" alt="Medtagger logo" />
```